Python_hw1/hw1.py

```diff
@@ -0,0 +1,20 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = str(input("Введите трехзначное число: "))
```

Comment:
Функция `input` уже возвращает строковый тип данных.
```suggestion
a = input("Введите трехзначное число: ")
```